src/main/java/com/karankumar/bookproject/ui/settings/SettingsView.java

```diff
@@ -47,10 +49,16 @@
 public class SettingsView extends HorizontalLayout {
 
     private static final String ENABLE_DARK_MODE = "Enable dark mode";
+    private static final String APPEARANCE = "Appearance:";
+    private static final String MYBOOKS = "My books:";
     private static final String DISABLE_DARK_MODE = "Disable dark mode";
     private static final SwitchToggle darkModeToggle;
     private static boolean darkModeOn = false;
     private static final Label darkModeLabel = new Label(ENABLE_DARK_MODE);
+    private static final H3 appearanceH3 = new H3(APPEARANCE);
+    private static final H3 myBooksH3 = new H3(MYBOOKS);
```

Comment:
Could you change the suffix to `Heading' instead of 'H3'? If we change the size of the heading, the variable name will not need changing